examples/tutorial_wikipedia_sessions.py

```diff
@@ -1,23 +1,32 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+from __future__ import absolute_import
+from builtins import next
+from future import standard_library
 import sys
+
 from traildb import TrailDB
 
+standard_library.install_aliases()
```

Comment:
Yes, it works without this. The `future` library help to keep backwards compatibility when use python 3 code, and at this moment is not necessary, I have added just to be safe in case we add some python 3.x code that can break backwards compatibility with python 2.x.

But to keep it simple, we can get rid of this.